plugin-server/tests/worker/ingestion/person-state.test.ts

```diff
@@ -25,17 +25,20 @@ describe('PersonState.update()', () => {
     let uuid2: UUIDT
     let teamId: number
     let poEEmbraceJoin: boolean
+    let organizationId: string
 
     beforeAll(async () => {
         ;[hub, closeHub] = await createHub({})
         await hub.db.clickhouseQuery('SYSTEM STOP MERGES')
+
+        organizationId = await createOrganization(hub.db.postgres)
```

Comment:
these tests were relatively slow and don't seem to need a new organisation per test

Reply:
yup I think new team is sufficient